.github/containers/test-installation/Dockerfile

```diff
@@ -14,5 +14,8 @@ RUN apt-get update -y && \
     python -m pip install --upgrade --no-cache-dir pip
 
 COPY dist dist
-RUN pip install dist/*.whl && \
-    rm -rf dist
+# This git-credentials file is made available by the GitHub ci.yaml workflow
+COPY git-credentials /root/.git-credentials
```

Comment:
Sorry for being late to the party. I only have a question.

IIUC the credentials will be stored in a intermediate layer of the docker image. This is not a security issue because the docker image is only used in the CI internally for testing, right? Just checking...

Reply:
ok, thanks for clarifying it